test/src/specs/tx_pool/valid_since.rs

```diff
@@ -236,11 +237,11 @@ impl ValidSince {
             absolute_block
         );
         (0..DEFAULT_TX_PROPOSAL_WINDOW.0).for_each(|_| {
-            node.generate_block();
+            mine(&node, 1);
```

Comment:
suggest to replace all `(0..DEFAULT_TX_PROPOSAL_WINDOW.0).for_each(|_| mine(node, 10))` with:
```suggestion
            mine(&node, DEFAULT_TX_PROPOSAL_WINDOW.0)
```

Reply:
Updated, thanks for your suggestion. Please take a look.